dsc_resource_release_tags.yml

```diff
@@ -0,0 +1,48 @@
+---
+xActiveDirectory: 2.12.0.0-PSGallery
+xAdcsDeployment: 1.0.0.0-PSGallery
+xAzure: 0.2.0.0-PSGallery
+xAzurePack: 1.4.0.0-PSGallery
+xBitlocker: 1.1.0.0-PSGallery
+xCertificate: 2.1.0.0-PSGallery
+xComputerManagement: 1.7.0.0-PSGallery
+xCredSSP: 1.1.0.0-PSGallery
+xDatabase: 1.4.0.0-PSGallery
+xDefender: 0.2.0.0-PSGallery
+xDFS: 3.1.0.0-PSGallery
+xDhcpServer: 1.4.0.0-PSGallery
+xDismFeature: 1.1.0.0-PSGallery
+xDnsServer: 1.7.0.0-PSGallery
+xExchange: 1.8.0.0-PSGallery
+xFailOverCluster: 1.4.0.0-PSGallery
+xHyper-V: 3.4.0.0-PSGallery
+xInternetExplorerHomePage: 1.0.0-PSGallery
+xJea: 0.3.0.0-PSGallery
+xMySql: 2.1.0.0-PSGallery
+xNetworking: 2.10.0.0-PSGallery
+xPendingReboot: 0.3.0.0-PSGallery
+xPhp: 1.2.0.0-PSGallery
+xPowerShellExecutionPolicy: 1.1.0.0-PSGallery
+xPSDesiredStateConfiguration: 3.12.0.0-PSGallery
+xRemoteDesktopAdmin: 1.1.0.0-PSGallery
+xRemoteDesktopSessionHost: 1.2.0.0-PSGallery
+xRobocopy: 2.0.0.0-PSGallery
+xSCDPM: 1.2.0.0-PSGallery
+xSCOM: 1.3.3.0-PSGallery
+xSCSMA: 1.3.0.0-PSGallery
+xSCSPF: 1.3.1.0-PSGallery
+xSCSR: 1.3.0.0-PSGallery
+xSCVMM: 1.2.4.0-PSGallery
+xSharePoint: 1.1.0.0-PSGallery
+xSmbShare: 1.1.0.0-PSGallery
+xSqlPs: 1.3.0.0-PSGallery
+xSQLServer: 1.7.0.0-PSGallery
+xStorage: 2.6.0.0-PSGallery
+xTimeZone: 1.5.0.0-PSGallery
+xWebAdministration: 1.12.0.0-PSGallery
+xWebDeploy: 1.2.0.0-PSGallery
+xWindowsEventForwarding: 1.0.0.0-PSGallery
+xWindowsRestore: 1.0.0-PSGallery
+xWindowsUpdate: 2.5.0.0-PSGallery
+xWinEventLog: 1.1.0.0-PSGallery
+xWordPress: 1.1.0.0-PSGallery
```

Comment:
@puppetlabs/windows may not be a bad idea for someone to go through and verify these are all correct. (and file tickets with Microsoft if they are not).